packages/letters/letters.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Client library for sending email over SMTP"
+description: "Simple to use SMTP client implementation for ocaml"
+maintainer: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+authors: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+homepage: "https://github.com/oxidizing/letters/"
+bug-reports: "https://github.com/oxidizing/letters/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
```

Comment:
Your `dune-project` file indicates this package requires at least dune 2.3

Reply:
I didn't realize that, thanks for pointing out.

Reply:
Fixed.

Reply:
Even though it looks correct on the branch (https://github.com/oxidizing/letters/blob/0.1.0/dune-project#L21), In the submitted version it is back to 2.0.

Reply:
@jerben pushed first wrong version and then moment after the correct one that should match with the release branch. So now you should have the correct one.